### PR TITLE
Feature/macgills/947 gradle to kotlin

### LINKED ISSUE #947: convert build.gradle to build.gradle.kts
gradle switched over from groovy to kotlin. please convert build.gradle, and settings.gradle to kotlin:
https://guides.gradle.org/migrating-build-logic-from-groovy-to-kotlin/
as documented there this does not need to be done at once.

### PR DESCRIPTION
**Pull request by macgills**
Fixes #947 

Convert gradle files to gradle.kts

### CODE REVIEW

**Comment by mhutti1** [comment]
I don't think this is related to this change but you used to be able to select customExample from flavour selection to build it. How do you build a custom app now in Android stuido?

**Comment by kelson42** [comment]
@mhutti1 The new whole process for custom app is documented here https://github.com/kiwix/kiwix-android-custom/blob/master/README.md.

**Comment by macgills** [comment]
@mhutti1 you should still be able to select `customExample`, I'll run a check on this branch but I was fairly sure I was still able to do it, the CI also lints customExample so I would have expected the build to fail if it couldn't find it

File: buildSrc/src/main/kotlin/Versions.kt [hunk 1]
```diff
@@ -41,29 +42,29 @@ object Versions {
...
 
   const val logging_interceptor: String = "3.12.1" // available: "4.2.2"
 
   const val ink_page_indicator: String = "1.3.0"
 
-  const val kotlin_stdlib_jdk7: String = "1.3.60"
+  const val kotlin_stdlib_jdk7: String = "1.3.61" // available: "1.3.61"
 
-  const val leakcanary_android: String = "2.0-alpha-2"
+  const val leakcanary_android: String = "2.0-alpha-2" // available: "2.0"
```
Comment (abdulwd) [inline buildSrc/src/main/kotlin/Versions.kt]: We can upgrade these to stable versions.

Comment (macgills) [reply buildSrc/src/main/kotlin/Versions.kt]: done

**Pull request merged**